README.md

```diff
@@ -138,6 +138,8 @@ Run `python workflow/scripts/flow_generator.py --help` for a list of options.
    
 ### Running the Cylc pipeline locally
 
+**Note:** Running the Cylc pipeline locally on a Mac will be siginificantly slower than running on Oscar due to Docker Desktop emulation. 
+
```

Comment:
Is it not possible to have a docker build that's specifically for arm64? It seems to me like this Stack Overflow answer suggests that you can avoid emulation by having a distinct build step: https://stackoverflow.com/questions/73938529/docker-multi-platform-builds-extremely-slow-for-arm64-on-gitlab-ci

Reply:
Thanks for starting this discussion @danielmwatkins! I had a similar thought and in Juila 1.9, there was a limitation that did not support `arm64` docker builds (https://github.com/JuliaLang/julia/issues/49474). But, I just updated to Julia 1.10 and it seems like the action built the docker for both platforms! I need to still test everything to find any bugs related to updating a major version in Julia.

Reply:
I am having trouble getting things to run from a Docker container with Julia 1.10. But, I am testing the end-end workflow on Oscar with Julia 1.10 and that is going well so far (still running). So I think all the IFT code is fine to use with Julia 1.10.